src/Toolbar.cpp

```diff
@@ -69,8 +69,8 @@ static ToolbarButtonInfo gToolbarButtons[] = {
     {5, CmdZoomFitPageAndSinglePage, _TRN("Fit a Single Page"), 0},
     {6, CmdZoomOut, _TRN("Zoom Out"), 0},
     {7, CmdZoomIn, _TRN("Zoom In"), 0},
-    {13, CmdViewRotateLeft, _TRN("Rotate Left"), 0},
-    {14, CmdViewRotateRight, _TRN("Rotate Right"), 0},
+    {13, CmdViewRotateLeft, _TRN("Rotate &Left\tCtrl+Shift+-"), 0},

```

Comment:
@kjk can correct me 
but I do not think this is the correct place for & shortcuts since they are provided elsewhere by _TRN

Reply:
I'm not sure, but without a full text I copied from `strings\translations.txt` the translation is not working 🤔

Reply:
I have to admit without comprehending  C++
 I do not understand all the ways the sections interact, but since translation works for other icons, I see no reason the extra ones do not follow UNLESS there is a limiting zone of action (but these are included amongst others) or index number limit (perhaps a 10 needs to be 12 ?)

Reply:
Other icons have no key shortcuts. I think is was much easier to include
key combination straight to a command name than creating second label next
to it.

Reply:
Other icons have no key shortcuts. I think is was much easier to include
key combination straight to a command name than creating second label next
to it.

Reply:
GitHubRulesOK: "The other icons should all have keyboard shortcuts (not always listed in menu) but their translations are functioning as tooltips without a need to show their controls. so French translation is visibly different but + & - keys work on QWERTY keyboard same as the + & - icons
Is it perhaps language specific? did you try switching to another language you recognise ? but note there is also [ & ] for rotation in addition to + & _ (note it is underline when shifted)"

Reply:
I see some redudancy e.g. in a translation of "Copy Selection" in `strings\translations.txt`. Once it's used in menu with a key shorctuct, next it's used in context menu(RMB) but without the key shortcut. Both have separate translaton in `strings\translations.txt` but it's basicly the same command. If I should remove the shortcut I have to create new translation without it.
Is see no keyboard shortucts on tooltips.

Reply:
Sorry for the mess. I replied by gmail 😞